projects/1.21/assets/occultism/occultism/lang/zh_cn.json

```
@@ -0,0 +1,3191 @@
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_random_animal.description.title": "召唤随机动物",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_random_animal.name": "被附身的随机动物",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_shulker.description.text": "在这个仪式中，一只[#](ad03fc)潜影贝[#]()将借由[#](ad03fc)史莱姆类生物[#]()的生命能量生成，然后立刻被召唤出的[#](ad03fc)火灵[#]()附身。[#](ad03fc)被附身的潜影贝[#]()被击杀后至少掉落一个[](item://minecraft:shulker_shell)。你可以使用原版的潜影贝增殖来产生普通的潜影贝，但是它们掉落的壳更少。\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_shulker.entity.text": "**掉落物**：1-2x[](item://minecraft:shulker_shell)\n并有10%%概率掉落一个[](item://minecraft:chorus_flower)\n或[](item://minecraft:spire_armor_trim_smithing_template);\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_shulker.name": "被附身的潜影贝",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_skeleton.description.text": "在这个仪式中，一只[#](ad03fc)骷髅[#]()利用[#](ad03fc)鸡[#]()的生命能量生成，然后立刻被召唤出的[#](ad03fc)魔精[#]()附身。[#](ad03fc)被附身的骷髅[#]()免疫日光伤害，而且被击杀后至少会掉落一个[](item://minecraft:skeleton_skull)。\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_skeleton.entity.text": "**掉落物**：1x[](item://minecraft:skeleton_skull)\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_skeleton.name": "被附身的骷髅",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_strong_breeze.description.text": "荒野强大旋风人为“流铸风形”，比普通的旋风人更为强大。它是获取[](item://minecraft:heavy_core)的最终目标，作为额外奖励，你可以获得：[](item://minecraft:flow_armor_trim_smithing_template)、[](item://minecraft:flow_banner_pattern)、[](item://minecraft:flow_pottery_sherd)和[](item://minecraft:music_disc_creator)。\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_strong_breeze.entity.text": "  **掉落物**：1x[](item://minecraft:heavy_core)以及其他掉落物（查看下一页）；\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_strong_breeze.name": "辉煌宝库",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_otherworld_bird.description.text": "请参阅[具翼使魔](entry://familiar_rituals/familiar_otherworld_bird)以获取更多信息。\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_otherworld_bird.entity.text": "**提供**：一只可驯服的具翼。\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_otherworld_bird.name": "未受缚的具翼",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_parrot.description.text": "在这个仪式中，一只[#](ad03fc)魔灵[#]()将会以**未驯服的魔灵**形式召唤。\n\\\n\\\n宰杀一只[#](ad03fc)鸡[#]()并提供染料，魔精将会获得鹦鹉的形态。由于[#](ad03fc)魔精[#]()并不是最聪明的魔灵，它们有时候会误解你的指示……\n",
+  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_parrot.description2.text": "*意思是说，如果生成了一只[#](ad03fc)鸡[#]()，这不是bug，只是运气不好！*\n",
```

```suggestion
  "book.occultism.dictionary_of_spirits.possession_rituals.possess_unbound_parrot.description2.text": "*意思是说，如果生成了一只[#](ad03fc)鸡[#]()，这不是Bug，只是运气不好！*\n",
```
前后Bug大小写统一，属于是硬抠中的硬抠